src/components/Header.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+
+import { COLORS } from '../constants/colors';
+
+import CustomStatusBar from './CustomStatusBar';
+import { GradientView } from './GradientView';
+
+type HomePropsType = {
+    children: React.ReactNode
+}
+
+export const Header = (props: HomePropsType) => {
+  const { children } =props;
+
+  return (
+    <View style={[styles.container]}>
+      <CustomStatusBar backgroundStyle={styles.backgroundStyle} barStyle={'light-content'} />
+      <GradientView colors={[COLORS.seaMariner, COLORS.dark]} style={styles.gradient}>
```

Comment:
Maybe it would have been better if you created a const array outside of the component so this array wouldnt recreate after each rerender